core/templates/pages/story-editor-page/editor-tab/story-editor.directive.html

```diff
@@ -4,9 +4,11 @@
       <label for="storyTitle" class="form-heading">Story Title</label>
       <input id="storyTitle" type="text" class="form-control protractor-test-story-title-field"
              ng-model="editableTitle" ng-blur="updateStoryTitle(editableTitle)"
-             placeholder="Enter a title for the story." maxlength="20" ng-trim="false">
-      <span class="form-text" style="font-size: smaller; color: red;" ng-if="editableTitle.length === 20">
-        <em>Please use at most 20 characters for the story title.</em>
+             placeholder="Enter a title for the story." maxlength="<[MAX_CHARS_IN_STORY_TITLE]>" ng-trim="false">
+      <span class="oppia-input-box-subtitle">
```

Comment:
Do we need the ` ng-if="editableTitle.length === 20"`, or we are going to present this all the time?

Reply:
It's going to be present always, since length at the limit is also valid and `maxlength` would prevent the user from typing more than the valid length anyway.